timemachine/potentials/nonbonded.py

```diff
@@ -129,7 +128,7 @@ def nonbonded_block_unsummed(
     lj = lennard_jones(dij, sig_ij, eps_ij)
 
     nrgs = jnp.where(dij < cutoff, es + lj, 0)
-    return nrgs
+    return cast(Array, nrgs)
```

Comment:
curiosity: What type is this casting from?

Reply:
The return type of `jnp.where` is `Array | tuple[Array, ...]`, I think because when `where` is called with only one argument the behavior is different. In this case, we know it will return an `Array`.